### DISCUSSION TITLE
Idea for “Open in App”

### DISCUSSION THREAD

**Discussion by yodaluca23** [qnblackcat/uYouPlus#668] [Ideas]
So couldn't someone theoretically publish an app to the Apple App Store (official) that can be set as your default browser, that when done so, any link opened from another app, would be sent to this app (because it's your "Browser"), and then (configurable in the app), the user could have it say, for example, open all youtube.com URLs in YouTube (sideloaded), so whenever it grabs that link because it's considered a web browser, it would then pass it off to YouTube, and then have it where if it's any other link not any of the ones that the user wants to be opened in a specific app to then pass it off to the user's real web browser of choice, again configurable in the app.

I don't know if this would be allowed on the App Store, I have no idea of Apple's rules on that kind of stuff, I've never tried this, and I'm not a Swift programer, but I think this could work if Apple allows it on the store.

I couldn't think of any better place to post this, so I'm posting it here, I thought about putting it on Reddit, but I am not sure so...

**Comment by qnblackcat** [comment]
https://github.com/qnblackcat/uYouPlus/wiki/FAQ#q-deep-link-aka-open-in-the-youtube-app-doesnt-work

**Reply by yodaluca23** [reply]
> https://github.com/qnblackcat/uYouPlus/wiki/FAQ#q-deep-link-aka-open-in-the-youtube-app-doesnt-work

I know about that, but none of those methods just open straight in YouTube, they all require more clicking... Other then TrollStore...

**Reply by qnblackcat** [reply]
So you want me to build an app that automatically opens YouTube link, and submit it to the AppStore?! 

Sorry but that’s beyond my knowledge

**Reply by yodaluca23** [reply]
> So you want me to build an app that automatically opens YouTube link, and submit it to the AppStore?! 
> 
> 
> 
> Sorry but that’s beyond my knowledge 

Not you, but just an idea for the community, maybe I should've put it on r/sideloaded instead.
